### PR TITLE
Add negative function to TensorFlow Frontend

### PR DESCRIPTION
**Pull request by kanakmi**
This PR addresses the issue #2712 which is tracked in #1545 

Apart from adding the negative function, I also arranged the previous functions alphabetically for faster navigation and better code readability.

### CODE REVIEW

**Comment by jkeane508** [comment]
Hi @kanakmi,

For this task you still have to implement the relevant test as exampled [here](https://lets-unify.ai/ivy/deep_dive/17_ivy_frontends_tests.html#ivy-frontend-tests) in the docs.

Once you have implemented this, please re-request a review from me :)

**Comment by kanakmi** [comment]
Hey @jkeane508 

I have added the tests as well. 
Can you please review the changes?

Thanks
Kanak

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py [hunk 1]
```diff
@@ -290,3 +290,29 @@ def test_tensorflow_logical_xor(
         x=np.asarray(x[0], dtype=input_dtype[0]),
         y=np.asarray(x[1], dtype=input_dtype[1]),
     )
+
+    
+# negative
+@given(
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=ivy_tf.valid_float_dtypes),
```
Comment (jkeane508) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py:297]: We can use `ivy_tf.valid_numeric_dtypes` as this function should be able to accept integer dtypes aswell as float

**Review by jkeane508 — CHANGES_REQUESTED** [review]
Just a small change to make but other than that looks good!

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py (hunk 1 above)
Comment (kanakmi) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py]: Fixed that @jkeane508

Comment (kanakmi) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py]: @jkeane508 
I should also add uint8, uint16, uint32, and uint64 to the unsupported types of negative right?

Comment (jkeane508) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py]: @kanakmi
Yep, makes sense :)

**Pull request closed without merging**